File: src/com/uipkg/components/defined/ClientMachine.java
package com.uipkg.components.defined;

import java.awt.Dimension;
import java.awt.Toolkit;

public final class ClientMachine {
  private ClientMachine() {}

  public static Dimension getClientScreenSpace() {
    return Toolkit.getDefaultToolkit().getScreenSize();
  }

  public static String getClientName() {
    return System.getProperty("os.name");
  }
}

File: src/com/uipkg/components/defined/ClientMachine.kt
package com.uipkg.components.defined

import java.awt.Dimension
import java.awt.Toolkit

object ClientMachine {
    val clientScreenSpace: Dimension
        get() = Toolkit.getDefaultToolkit().screenSize
    val clientName: String
        get() = System.getProperty("os.name")
}